code/modules/mob/living/simple_animal/hostile/nanotrasen.dm

```diff
@@ -24,15 +24,18 @@
 	loot = list(/obj/effect/mob_spawn/human/corpse/nanotrasensoldier)
 	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
 	unsuitable_atmos_damage = 15
-	faction = list("nanotrasenprivate")
+	faction = list(ROLE_DEATHSQUAD)
+	check_friendly_fire = 1
 	status_flags = CANPUSH
-	speak = list("Stop resisting!", "I AM THE LAW!", "Face the wrath of the golden bolt!", "Stop breaking the law, asshole!")
-	search_objects = 1
-
+	del_on_death = 1
```

Comment:
```suggestion
	del_on_death = TRUE
```

Reply:
By the way, there is a difference from "1" and "true"?